.github/CODEOWNERS

```diff
@@ -2492,12 +2487,15 @@ x-pack/solutions/security/plugins/security_solution/public/asset_inventory @elas
 x-pack/solutions/security/plugins/security_solution/public/siem_readiness @elastic/contextual-security-apps
 
 ## Security Solution sub teams - Security Entity Store
-x-pack/platform/packages/shared/kbn-entities-schema/src/schema/v1 @elastic/core-analysis
-x-pack/platform/plugins/shared/entity_manager/server/lib/entities @elastic/core-analysis
-x-pack/platform/plugins/shared/entity_manager/server/lib/auth @elastic/core-analysis
+x-pack/platform/packages/shared/kbn-entities-schema @elastic/core-analysis
+x-pack/platform/plugins/shared/entity_manager @elastic/core-analysis
+x-pack/platform/test/api_integration/apis/entity_manager/fixture_plugin @elastic/core-analysis
```

Comment:
we might not need this since `x-pack/platform/test/api_integration/apis/entity_manager` came to us already